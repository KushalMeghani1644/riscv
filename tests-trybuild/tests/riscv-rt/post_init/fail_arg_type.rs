#[riscv_rt::post_init]
fn before_main(_hart_id: String) {}

fn main() {}
