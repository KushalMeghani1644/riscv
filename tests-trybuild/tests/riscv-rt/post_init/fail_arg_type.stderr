error: argument type must be usize
 --> tests/riscv-rt/post_init/fail_arg_type.rs:2:26
  |
2 | fn before_main(_hart_id: String) {}
  |                          ^^^^^^
